app/dashboard/views.py

```diff
@@ -924,6 +927,282 @@ def save_search(request):
+    total_contributors_count = 142
+
+    total_paid_dollars = 0
+    total_paid_eth = 0
+
+    for bounty in done_bounties:
+        bounty_value_in_usdt = bounty.get_value_in_usdt
+        bounty_value_in_eth = bounty.get_value_in_eth
+
+        if bounty_value_in_usdt is not None:
+            total_paid_dollars = total_paid_dollars + bounty.get_value_in_usdt
+
+        if bounty_value_in_eth is not None:
+            total_paid_eth = total_paid_eth + bounty.get_value_in_eth
+
+    # TODO: total paid date since - date of first submitted bounty
+    total_paid_date_since = _("May 5. 2018")
+
+    # total budget in dollars, not used currently, we show an input field by default
+    total_budget_dollars = ""
+
+    # total budget in eth, not used currently, we show an input field by default
+    total_budget_eth = ""
+
+    # TODO: Tax year stats - Considering the tax year to be 1 Jan to 31 Dec
+    tax_year = 2018
+    tax_year_bounties_count = 139
+    tax_year_bounties_worth_dollars = 34.500
+
+    # Latest on your bounties
+    # TODO: What's the difference between an 'issue' and a 'bounty'? Just using bounty for these right now..
+    expired_issues_count = expired_bounties.count()
+    expired_issues_worth_dollars = 0
+    for expired_issue in expired_bounties:
+        issue_worth_in_usdt = expired_issue.get_value_in_usdt
+        if issue_worth_in_usdt is not None:
+            expired_issues_worth_dollars = expired_issues_worth_dollars + issue_worth_in_usdt
+
+    active_bounties_count = active_bounties.count()
+    completed_bounties_count = done_bounties.count()
+    expired_bounties_count = expired_bounties.count()
+
+    outgoing_funds_filters = [
+        {
+            'value': 'All',
+            'value_display': _('All'),
+            'is_all_filter': True,
+            'is_type_filter': False,
+            'is_status_filter': False
+        },
+        {
+            'value': 'Tip',
+            'value_display': _('Tip'),
+            'is_all_filter': False,
+            'is_type_filter': True,
+            'is_status_filter': False
+        },
+        {
+            'value': 'Payment',
+            'value_display': _('Payment'),
+            'is_all_filter': False,
+            'is_type_filter': True,
+            'is_status_filter': False
+        },
+        {
+            'value': 'Pending',
+            'value_display': _('Pending'),
+            'is_all_filter': False,
+            'is_type_filter': False,
+            'is_status_filter': True
+        },
+        {
+            'value': 'Claimed',
+            'value_display': _('Claimed'),
+            'is_all_filter': False,
+            'is_type_filter': False,
+            'is_status_filter': True
+        },
+    ]
+
+    outgoing_funds = []
+    for bounty in done_bounties.filter(fulfillment_started_on__isnull=False, fulfillment_submitted_on__isnull=True):
+        # TODO: Need the txid to generate this link. Where is it in the bounty object, if at all?
```

Comment:
https://github.com/gitcoinco/web/pull/1722 

adds the ability to pull txids.  you can expect that on a go-forward basis the txids will be there